### ISSUE TITLE
[Bug]: plugin won't start after update to 1.5.0-71

### ISSUE THREAD

**Issue by 506V** [t0bst4r/matterbridge-home-assistant-addon#84]
### What happened?

The plugin is not starting after update

### Relevant log output

```shell
[20:52:16.989] [Matterbridge] Added and registered bridged device (1/1) Home Lights  (MA-onofflight) for plugin matterbridge-home-assistant 
file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/device/Device.js:164
    throw new ImplementationError("createOptionalClusterServer needs to be implemented by derived classes");
          ^

ImplementationError: createOptionalClusterServer needs to be implemented by derived classes
    at MatterbridgeDevice.createOptionalClusterServer (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/device/Device.js:164:11)
    at MatterbridgeDevice.getClusterServer (file:///usr/local/lib/node_modules/matterbridge/node_modules/@project-chip/matter.js/dist/esm/device/Device.js:176:37)
    at get levelControlCluster [as levelControlCluster] (file:///usr/local/lib/node_modules/matterbridge-home-assistant/lib/devices/aspects/level-control-aspect.js:8:28)
    at LevelControlAspect.update (file:///usr/local/lib/node_modules/matterbridge-home-assistant/lib/devices/aspects/level-control-aspect.js:32:48)
    at LightDevice.updateState (file:///usr/local/lib/node_modules/matterbridge-home-assistant/lib/devices/home-assistant-device.js:21:26)
    at async HomeAssistantMatterAdapter.create (file:///usr/local/lib/node_modules/matterbridge-home-assistant/lib/home-assistant/home-assistant-matter-adapter.js:91:9)
    at async HomeAssistantMatterAdapter.expensiveUpdate (file:///usr/local/lib/node_modules/matterbridge-home-assistant/lib/home-assistant/home-assistant-matter-adapter.js:45:13)
    at async HomeAssistantClient.expensiveUpdate (file:///usr/local/lib/node_modules/matterbridge-home-assistant/lib/home-assistant/home-assistant-client.js:50:13)

Node.js v18.20.3
s6-rc: info: service legacy-services: stopping
s6-rc: info: service legacy-services successfully stopped
s6-rc: info: service legacy-cont-init: stopping
s6-rc: info: service legacy-cont-init successfully stopped
s6-rc: info: service fix-attrs: stopping
s6-rc: info: service base-addon-log-level: stopping
s6-rc: info: service fix-attrs successfully stopped
s6-rc: info: service base-addon-log-level successfully stopped
s6-rc: info: service base-addon-banner: stopping
s6-rc: info: service base-addon-banner successfully stopped
s6-rc: info: service s6rc-oneshot-runner: stopping
s6-rc: info: service s6rc-oneshot-runner successfully stopped
```

### Version

 Add-on version: 1.5.0-71  You are running the latest version of this add-on.  System: Home Assistant OS 12.4  (amd64 / generic-x86-64)  Home Assistant Core: 2024.6.4  Home Assistant Supervisor: 2024.06.2

**Comment by t0bst4r** [comment]
Is `Home Lights` a light WITHOUT brightness control?

**Comment by t0bst4r** [comment]
Could you try the new bug fix release 1.5.1?

**Comment by 506V** [comment]
> Is `Home Lights` a light WITHOUT brightness control?

You were right. I disabled the problematic helper and add-on starter.

**Comment by t0bst4r** [comment]
Yeah it was not a problem with your light, but a bug with my latest change. I just wanted to verify the bug. Should be fixed with 1.5.1

**Comment by 506V** [comment]
> Could you try the new bug fix release 1.5.1?

This update worked too (with problematic helper enabled)
